internal/olm/operator/registry/configmap/deployment.go

```diff
@@ -28,7 +28,7 @@ const (
 	// The image operator-registry's initializer and registry-server binaries are run from.
 	// This iamge has been pinned to the latest version containing the initializer and registry-server
 	// binaries, which were deprecated in https://github.com/operator-framework/operator-registry/pull/587.
-	registryBaseImage = "quay.io/operator-framework/upstream-registry-builder:v1.16.0"
+	registryBaseImage = "quay.io/operator-framework/opm:latest"
```

Comment:
Do we want latest or should we be pinning?

Reply:
I think pinning to latest is fine. For the previous image, we had to pin it to a specific version as the `initializer` and `registry-server` were being deprecated.

Reply:
@rashmigottipati where/how/why this image is used? Could you share here if it will be like an opm command what would be the command? Is that used as a param of binary-image option? Could we also supplement the comment above adding this info?